conf/v0.6/general/test/llama.toml

```diff
@@ -0,0 +1,12 @@
+name = "llama"
+description = "Llama2 70b"
+test_template_name = "NeMoLauncher"
+# FIX ME : ~training.model.position_embedding_type is a quick fix that should be changed with newer version of container
```

Comment:
Could you please explain this in detail in the additional section of the PR summary? I don't understand this issue and why you have added this note. In addition, please rename "FIX ME" to "FIXME" so that we can easily identify this keyword later.

Reply:
The comment is actually not clear. Another contributor may not understand which field to update. Please clearly specify which field of which file should be updated to fix the issue.

Reply:
So as far as I remember, there was a bug in loading the latest release of the container with Llama, consulting with Zsolt, adding this : ~training.model.position_embedding_type made it work.
In the newest releases, we shouldnt need this fix as they would hopefully solve the bug

Reply:
cc: @srinivas212

Reply:
I'll update the PR for clarification

Reply:
@TaekyungHeo Fixed. please resolve

Reply:
It is not fixed yet. Could you please use FIXME instead of "FIX ME" so that we can track FIXMEs or TODOs with grep?